plugins/plugin-site/src/templates/search.jsx

```diff
@@ -61,8 +61,8 @@ const doSearch = (data, setResults, categoriesMap) => {
             plugins: hits
         });
     }).catch(err => {
-        window?.Sentry?.captureException(err);
-        // FIXME alert/console.log/somehow tell user something went wrong
+        /* eslint no-console: ["error", { allow: ["log"] }] */
```

Comment:
```suggestion
        // eslint-disable-next-line no-console
```

I think the other one modifies config for entire file.

Reply:
Done